### DISCUSSION TITLE
write attributes like stroke-width as inline

### DISCUSSION THREAD

**Discussion by ChristinaFaur** [vvvv/SVG#852] [Q&A]
if stroke-width is set for a SvgRectangle it is not displayed as "inline" style

this is how I want to display

![image](https://user-images.githubusercontent.com/63300027/79142261-83b11b80-7dbb-11ea-9643-803530becd2c.png)

this is how it is displayed when writing svg-File:
![image](https://user-images.githubusercontent.com/63300027/79142584-0508ae00-7dbc-11ea-9013-7f9721fce6cf.png)

Is possible to achieve this?

**Comment by H1Gdev** [comment]
Only properties in `style` are `fill` and `stroke`.

**Comment by ChristinaFaur** [comment]
Thanks for your answer. Is there a reason why other properties not included are?

**Comment by H1Gdev** [comment]
I don't know reason why.(Why included only `fill` and `stroke` in `style`)
